main/sdr.go

```diff
@@ -147,12 +149,14 @@ func getPPM(serial string) int {
 
 func (e *ES) sdrConfig() (err error) {
```

Comment:
we created our own Go wrapper around the dump978 tool (and compile it as a library) so we're able to configure the dongle internally (using gortlsdr) whereas we call the external dump1090 binary via exec.Command (shell command) and pass it options which it uses to configure the dongle.